.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,16 +5,28 @@
 ## Provide some background on the changes
 > Give details ex. Security patching, content update, more API pods etc
 
-## If you are releasing a new version of notify, what components are you updating
+## If you are releasing a new version of Notify, what components are you updating
 - [ ] API
 - [ ] Admin
 - [ ] Documentation
-- [ ] Document API
-- [ ] Document UI
+- [ ] Document download API
+- [ ] Document download frontend
 
 ## Checklist if releasing new version:
```

Comment:
Do we want to add a point that is related to our ZenHub process? To make sure the task is set to done and approved by QA? or at least in the QA column (realistically, I am not sure people will have time to coordinate every day)?

Reply:
Are we going to have some sort of daily card for releases? I don't think we have one now.

Reply:
No we don't, I am not sure if it will be useful, i.e. if folks will look at our zendesk board than the Slack notification for example. 😅